metadatastore/odm_templates.py

```diff
@@ -184,4 +165,4 @@ class Event(Document):
     # TODO validate on this better
     data = DictField(required=True)
     time = FloatField(required=True)
-    meta = {'indexes': ['-descriptor', '-_id', '-uid'], 'db_alias': ALIAS}
+    meta = {'indexes': ['descriptor', '-time'], 'db_alias': ALIAS}
```

Comment:
`_id` is automatically indexed (because mongo) as is `uid` (because it is unique).  time needs to be indexed so that the sorts don't blow up, although I am now confused by the direction of time here.